docs/tuning-guide.md

```diff
@@ -210,6 +210,13 @@ rather than megabytes or smaller.
 Note that the GPU can encode Parquet and ORC data much faster than the CPU, so the costs of
 writing large files can be significantly lower.
 
+Suggest use Hive Parquet or ORC tables instead of Hive Text table as the intermediate tables for
+CTAS(`Create Table AS`) queries. The suggested change is to add `stored as parquet` for CTAS.
```

Comment:
```suggestion
Use Hive Parquet or ORC tables instead of Hive Text table as the intermediate tables for
CTAS(`Create Table As Select`) queries. The suggested change is to add `stored as parquet` for CTAS.
```

Reply:
Manually fixed this. @jlowe  want to take a look again?